.github/workflows/scanner-build.yaml

```diff
@@ -46,14 +46,14 @@ jobs:
         condensed="$(jq -c <<< "$matrix")"
         echo "matrix=$condensed" >> "$GITHUB_OUTPUT"
 
-  build-and-push:
-    needs: define-job-matrix
+  build-and-push-scanner:
```

Comment:
Maybe also change the id of `define-job-matrix` step below?

Reply:
I believe that was already done? I did catch other instances, though

Reply:
Ah, yeah, I meant "above".